opentreemap/importer/models/base.py

```diff
@@ -30,6 +31,15 @@ class Meta:
     FAILED_FILE_VERIFICATION = 6
     CANCELED = 8
     VERIFICATION_ERROR = 10
+    UNKNOWN_ERROR = 12
```

Comment:
Since this represents the specific case where the celery task disappears without completing I think it would be clearer to call it something like `TASK_VANISHED`. People reading the code would have an idea what it's about, while `UNKNOWN_ERROR` could mean many things.